demos/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ macro(CREATE_DEMO _target _sources)
     target_link_libraries(${_target}
         freetype-gl
         ${OPENGL_LIBRARY}
-        ${GLFW3_LIBRARY}
+        glfw ${GLFW_LIBRARIES}
```

Comment:
Adrian, thanks for the my patch checkup.

>>Also why does this contain GLFW3_LIBRARY AND GLFW_LIBRARIES. The tools only support GLFW3.
It does not containg GLFW3_LIBRARY AND GLFW_LIBRARIES.
GLFW3 tarket_link_libraries is done like in this link:
http://www.glfw.org/docs/3.1/build.html

>>Useless, glfw is either contained in GLFW_LIBRARIES or not.
In Ubuntu if i only specify GLFW_LIBRARIES , it does not link.
i have to add glfw GLFW_LIBRARIES , as specified on their website.
>>Doesn't contain checks for existance, adds useless statements and breaks on some platforms.
They are not useless , they make the software compilable on Linux(which I believe it's the main platform for this program). 
I didn't test it on Windows , it is unfortunately it it breaks , i will check it .